generators/server/index.js

```diff
@@ -402,11 +403,11 @@ module.exports = JhipsterServerGenerator.extend({
 
         let logMsg =
             'Run your Spring Boot application:' +
-            '\n ' + chalk.yellow.bold('./mvnw');
+            '\n ' + chalk.yellow.bold('.' + path.sep + 'mvnw');
```

Comment:
My test results where this was working:
* in Windows cmd.exe produced `.\mvnw` and this works
* in Windows PowerShell produced `.\mvnw` and this works (in PowerShell also `./mvnw` works)
* in Windows Ubuntu bash produced `./mvnw` and this works
* in Ubuntu produced `./mvnw` and this works

And in this case this was not working:
* in Windows git bash produced `.\mvnw` and this is not working, in Windows git bash are working `./mvnw` and `.\\mvnw`

So this solution is incomplete. Currently I have no idea how to achieve correct result. Possible solutions seem to be:
1. close this PR in favor to Windows git bash
2. merge this PR in favor to Windows cmd.exe